documents/entity/service_test.go

```diff
@@ -0,0 +1,362 @@
+// +build unit
+
+package entity
+
+import (
+	"github.com/centrifuge/centrifuge-protobufs/gen/go/entity"
+	"github.com/centrifuge/go-centrifuge/documents"
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/storage"
+	"github.com/centrifuge/go-centrifuge/testingutils/documents"
+	"github.com/centrifuge/go-centrifuge/utils"
+
+	cliententitypb "github.com/centrifuge/go-centrifuge/protobufs/gen/go/entity"
+	"github.com/centrifuge/go-centrifuge/testingutils"
+	"github.com/centrifuge/go-centrifuge/testingutils/commons"
+	"github.com/centrifuge/go-centrifuge/testingutils/config"
+
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/testingutils/identity"
+	"github.com/centrifuge/go-centrifuge/transactions"
+	"github.com/centrifuge/gocelery"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func getServiceWithMockedLayers() (testingcommons.MockIdentityService, Service) {
```

Comment:
maybe we can think about moving this helper function to a utils package because it is also being redeclared and used in the invoice and PO service_test.go